### DISCUSSION TITLE
Small bug when using "returning: false"

### DISCUSSION THREAD

**Discussion by Camarade84** [sequelize/sequelize#14131] [Help & Questions]
when I try to create a record using the model
`this.ModelName.create(creationOptions, {returning: false})`

I get an error on the output, the record to the database is successful
`TypeError: Cannot read property 'id' of undefined`

made changes to the file to fix
/src/dialects/mssql/query.js:181

```
if (this.isInsertQuery(data)) {

this.handleInsertQuery(data);

      return [this.instance || data, rowCount];
}
```
to 
```
if (this.isInsertQuery(data)) {
    if (this.options.returning) {
        this.handleInsertQuery(data);
    }
return [this.instance || data, rowCount];
}
```
Perhaps there is an easier way to solve this problem?

**Comment by ephys** [comment]
Could you open a new bug report with a SSCCE that reproduces the bug?
